lte/gateway/docker/mme/Dockerfile.rhel8

```diff
@@ -344,8 +344,7 @@ RUN cd json && \
     cd _build && \
     cmake3 -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Release .. && \
     make -j`nproc` && \
-    make install && \
-    cp /usr/local/include/nlohmann/json.hpp /usr/local/include/
```

Comment:
@themarwhal  Marie

This modification is not tested in the OAI CI pipeline. Only when your PR is `merged`. But it should be OK.

Ping me when it is merged. I will also have to recreate a base image with this modification.

Regards,
Raphael

Reply:
Is there a way to ensure nothing breaks before merging?

Reply:
Let me try manually

Reply:
@themarwhal Marie

OK for me. You can merge!